packages/shared/src/hooks/chat/useChatStream.ts

```diff
@@ -40,7 +42,11 @@ export const useChatStream = (): UseChatStream => {
 
       setSessionId(null);
 
-      let queryKey: ReturnType<typeof generateQueryKey> = null;
+      let queryKey: ReturnType<typeof generateQueryKey> = generateQueryKey(
+        RequestKey.Search,
+        user,
+        Date.now().toString(),
```

Comment:
Added this to have a default unique query key in case of any initial errors. Thise one will get overwritten in `session_created` immediately for usual queries.